addons/arsenal/functions/fnc_addVirtualItems.sqf

```diff
@@ -47,25 +47,27 @@ private _cargo = _object getVariable [QGVAR(virtualItems), [
 
 private _configCfgWeapons = configFile >> "CfgWeapons"; //Save this lookup in variable for perf improvement
 
-if (_items isEqualType true && {_items}) then {
-    private _configItems = uiNamespace getVariable QGVAR(configItems);
+if (_items isEqualType true) then {
+    if (_items) then {
```

Comment:
I don't see the improvement here :D

Reply:
passing false would error out since it would try to execute the else part, now it just silently end.

Reply:
Ah didn't see the else 👍